lib/remote/httphandler.cpp

```diff
@@ -112,7 +113,10 @@ void HttpHandler::ProcessRequest(
 				break;
 			}
 		}
-	} catch (const std::exception&) {
+	} catch (const std::exception& ex) {
+		Log(LogWarning, "HttpServerConnection")
+			<< "Permission error while processing HTTP request: " << ex.what();
```

Comment:
As you're catching `std::exception` here, this could possibly be anything, not only a permission problem. I wouldn't consider the exception from your example log as a permission error:

```
[2020-07-06 14:34:46 +0200] warning/HttpServerConnection: Permission error while processing HTTP request: Cannot convert value of type 'String' to an object.
```

So either go with a more generic message here (as that's all we know at this point) or if there are any meaningful more specific exception classes to catch, you could have multiple catch blocks with different messages.